payments/src/main/java/it/gov/pagopa/payments/controller/receipt/IPaymentsController.java

```diff
@@ -50,8 +50,6 @@ ResponseEntity<String> getReceiptByIUV(
     @ApiResponses(value = {
             @ApiResponse(responseCode = "200", description = "Obtained all organization payment positions.", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = ReceiptsInfo.class))),
             @ApiResponse(responseCode = "401", description = "Wrong or missing function key.", content = @Content(schema = @Schema())),
-            @ApiResponse(responseCode = "404", description = "No receipts found.", content = @Content(schema = @Schema(implementation = ProblemJson.class))),
```

Comment:
why removed 404 ?

Reply:
As agreed we no longer return that kind of error externally. In the worst case scenario, the client will receive an empty list